packages/build/src/js-transform.ts

```diff
@@ -213,6 +223,17 @@ export function jsTransform(js: string, options: JsTransformOptions): string {
         throw e;
       }
     }
+
+    if (options.transformModulesToAmd === true ||
+        (options.transformModulesToAmd === 'auto' &&
+         ast.program.sourceType === 'module')) {
+      doBabelTransform = true;
+      plugins.push(...babelTransformModulesAmd);
+    }
+
+    if (doBabelTransform) {
+      js = babelCore.transformFromAst(ast, js, {presets, plugins}).code!;
```

Comment:
`code!` looks suspicious here. The library is telling us that it could be null, and we're overriding it. If the library is right, we're kicking the exception down the road to a less clear place.

WDYT about checking and throwing instead?

Reply:
Done. I doubt it can happen without Babel itself throwing, probably just a weakness in the Babel typings (the thing it returns is used by a bunch of functions and has optional everything https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/babel-core/index.d.ts#L172), but I can't say for sure, so added a check and throw.